src/data/VariantSelector.ts

```diff
@@ -167,6 +168,49 @@ export function isValidPangoLineageQuery(query: string): boolean {
   return /^([A-Z]){1,3}(\.[0-9]{1,3})*(\.?\*)?$/.test(query.toUpperCase());
 }
 
+const stripNumber = (str: string) => Number.parseInt(str.replace(/\D+/g, ''));
+const isNumeric = (str: string) => /^\d+$/.test(str);
+
+type geneType = {
```

Comment:
Types should start with an upper case and not contain "type" -> just name it `Gene`